python/test/t_ProjectionStrategy_std.expout

```diff
@@ -0,0 +1,164 @@
++ Compute flood model by regression
+leastSquaresStrategy
+ProjectionStrategyImplementation
+- coefficients: 0
+- residual: 0
+- relative error: 0
+- measure: Distribution
+- weighted experiment: WeightedExperiment
+- input sample: size= 0 x dimension= 0
+- output sample: size= 0 x dimension= 0
+- weights: dimension= 0
+- design: size= 0
```

Comment:
same here, why 0?

Reply:
This is because the printed projection strategy was the original projection, and not the projection from the algorithm. Fixed at c9944a464d85017cec554890baa6874ceea4b516.